bin/grep-applab-source

```diff
@@ -44,20 +42,34 @@ base_dir = CDO.sources_s3_directory
 count = 0
 key_not_found = 0
 
-puts "Fetching a list of applab apps from Pegasus DB. This step may take approximately 10 minutes"
-puts "to run over ~3M rows in storage_apps in production..."
+# set this to a value like 1.year.ago or '2016-10-15' to search only recent
+# projects. This can be useful to search the remaining projects if the first run
+# dies part way through.
+created_after = nil
+
+puts "[#{Time.now}] Fetching a list of applab apps from Pegasus DB. This step may take approximately 20 minutes"
```

Comment:
nit: Any reason this can't be combined into one `puts` statements, perhaps with an EOS?